.github/workflows/deploy.yml

```diff
@@ -67,9 +67,9 @@ jobs:
       # Extract branch name
       - name: Extract branch name
         shell: bash
-        ## Cut off "refs/heads/" and only allow alphanumeric characters,
+        ## Cut off "refs/heads/", only allow alphanumeric characters and convert to lower case,
         ## e.g. "refs/heads/features/hello-1.2.0" -> "features_hello_1_2_0"
-        run: echo "branch=$(echo $GITHUB_HEAD_REF | sed 's/refs\/heads\///' | sed 's/[^a-z0-9]/_/ig')" >> $GITHUB_OUTPUT
+        run: echo "branch=$(echo $GITHUB_HEAD_REF | sed 's/refs\/heads\///' | sed 's/[^a-z0-9]/_/ig') | sed 's/[A-Z]/\L&/g'" >> $GITHUB_OUTPUT
```

Comment:
Is this a requirement for this PR?

Reply:
Yes I needed it as this PR also contained camel case.